### PR TITLE
add get sub-command

### PR DESCRIPTION
**Pull request by peb-adr**

### CODE REVIEW

File: pkg/datastorereader/datastorereader.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+			"filter": {
+				"field": "%s",
+				"value": "%s",
+				"operator": "="
+			}%s
+		}`,
+		collection, key, value, fieldsStr,
+	)
+	addr := d.readerURL.String() + filterSubpath
+	fmt.Printf("req: %v\n", reqBody)
```
Comment (normanjaeckel) [inline pkg/datastorereader/datastorereader.go:83]: Remove this line.

File: pkg/get/get.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+	// GetHelpExtra contains the long help text for the command without
+	// the headline.
+	GetHelpExtra = `Provide a collection to list contained models.
+Use options to narrow down output.`
+)
+
+// Cmd returns the get subcommand.
+func Cmd(cmd *cobra.Command, cfg connection.Params) *cobra.Command {
+	cmd.Use = "get"
```
Comment (normanjaeckel) [inline pkg/get/get.go:26]: Should be `cmd.User = "get collection"`

File: pkg/get/get.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+	cmd.Short = GetHelp
+	cmd.Long = GetHelp + "\n\n" + GetHelpExtra
+	cmd.Args = cobra.ExactArgs(1)
+
+	existsHelpText := "check only for existance (requires --filter)"
+	exists := cmd.Flags().Bool("exists", false, existsHelpText)
+
+	filterHelpText := "provide a filter based on a collection field"
```
Comment (normanjaeckel) [inline pkg/get/get.go:34]: maybe provide a short example here so that one uses the x=y syntax

**Review by normanjaeckel — APPROVED** [review]
Looks good, only some small remarks.

**Comment by peb-adr** [comment]
@normanjaeckel 
I tested both `get` and `set` now after the rebase.
Maybe you could run `make protoc` on your end again and see if the generated files differ. I got the feeling they will, because we have slightly different setups.
So you can either push your proto files on top of this, or just merge at will.

**Comment by peb-adr** [comment]
> @normanjaeckel I tested both `get` and `set` now after the rebase. Maybe you could run `make protoc` on your end again and see if the generated files differ. I got the feeling they will, because we have slightly different setups. So you can either push your proto files on top of this, or just merge at will.

@normanjaeckel could you merge this one or the other way this weekend?

**Pull request merged**